PVEntityGenerator/xml/PVEntityGenerator.xml

```diff
@@ -1,4 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!-- $URL$ -->
```

Comment:
please do not add this, it's SVN-specific

Reply:
ok, removed